src/main/java/uk/gov/hmcts/ccd/domain/model/casedataaccesscontrol/AccessProfile.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.ccd.domain.model.casedataaccesscontrol;
+
+public class AccessProfile {
+
+    private Boolean readOnly;
+    private String classification;
```

Comment:
```suggestion
    private String securityClassification;
```

Reply:
Furthermore, should we use the `uk.gov.hmcts.ccd.data.casedetails.SecurityClassification` enum